src/Microsoft.AspNetCore.Routing/RouteOptions.cs

```diff
@@ -10,10 +10,15 @@ namespace Microsoft.AspNetCore.Routing
     public class RouteOptions
     {
         /// <summary>
-        /// Gets or sets a value indicating whether all generated URLs are lower-case.
+        /// Gets or sets a value indicating whether all generated URLs are lower-case. it doesn't include the query string.
```

Comment:
Suggestions for these:

`Gets or sets a value indicating whether all generated paths URLs are lower-case. Use <see cref="LowercaseQueryStrings" /> to configure the behavior for query strings.`